### PR TITLE
Added algorithm for DC electric power formula

### PR DESCRIPTION
**Pull request by joseph-alan-jose**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: physics/electric_power.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+"""
+Calculating Electric Power
+Description : Electric power can be said to be the rate at which
```
Comment (rohan472000) [inline physics/electric_power.py:3]: The provided code calculates the electric power for DC circuits based on the given input parameters, which include the potential difference (voltage), current, and resistance. 

You should change PR name as DC Electric Power formula.

**Review by rohan472000 — CHANGES_REQUESTED** [review]

File: physics/electric_power.py (hunk 1 above)
Comment (joseph-alan-jose) [reply physics/electric_power.py:3]: Thankyou for the feedback

**Review by rohan472000 — APPROVED** [review]

**Review by rohan472000 — CHANGES_REQUESTED** [review]
change the function name to dc_power

File: physics/electric_power.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+P = I²R = V²/R
+
+Where, R refers to resistance
+
+reference: https://en.wikipedia.org/wiki/Electric_power
+
+"""
+
+
+def power(**values: float) -> float:
```
Comment (ChrisO345) [inline physics/electric_power.py:33]:
```suggestion
def dc_power(**values: float) -> float:
```

**Review by ChrisO345 — CHANGES_REQUESTED** [review]

File: physics/electric_power.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    # function accepts two arguments from a list of three
+    """
+    >>> power(potential_difference=5.0, current=6.0)
+    30.0
+    >>> power(current=2.0, resistance=5.0)
+    20.0
+    >>> power(potential_difference=4.0, resistance=2.0)
+    8.0
```
Comment (ChrisO345) [inline physics/electric_power.py:41]:
```suggestion
    >>> dc_power(potential_difference=5.0, current=6.0)
    30.0
    >>> dc_power(current=2.0, resistance=5.0)
    20.0
    >>> dc_power(potential_difference=4.0, resistance=2.0)
    8.0
```

File: physics/electric_power.py (hunk 2 above)
Comment (ChrisO345) [inline physics/electric_power.py:33]:
```suggestion
def dc_power(**values: float) -> float:
```

**Review by ChrisO345 — CHANGES_REQUESTED** [review]

**Comment by tianyizheng02** [comment]
Thanks for your contribution, but we already have an implementation of the electric power formula in [electronics/electric_power.py](https://github.com/TheAlgorithms/Python/blob/f8fe72dc378232107100acc1924fef31b1198124/electronics/electric_power.py). However, that implementation doesn't support calculating resistance, so you're free to build upon that implementation and extend its functionality.

**Pull request closed without merging**